text/0100-sensitive-data-handling.md

```diff
@@ -0,0 +1,103 @@
+#  Sensitive Data Handling
+
+By default, OpenTelemetry libraries never capture potentially-sensitive data.
```

Comment:
This would mean that not even URLs can be captured.

Reply:
It's a fair point - how does "... never capture potentially-sensitive data, except for the full URL" sound?